src/components/Contribute/Skill/index.tsx

```diff
@@ -16,7 +16,7 @@ import { getGitHubUsername } from '../../../utils/github';
 import { useSession } from 'next-auth/react';
 import YamlCodeModal from '../../YamlCodeModal';
 import { AttributionData, SchemaVersion, SkillYamlData } from '@/types';
-import SkillDescription from './SkillDescription';
+import SkillDescription from './SkillDescription/SkillDescriptionContent';
```

Comment:
This is not the correct component to import. You have imported just the content. The 'wrapper' component SkillDescription is what should be imported here. 

If you run `npm run dev` you can see that the Expandable section is not being rendered.

Compare the your branches UI to the main branch UI and this should make sense.

Reply:
thank you for bringing this to attention, the rendering issue has been resolved.